### DISCUSSION TITLE
b455ea0e-0b1c-41e3-a97a-2e2315b20d08

### DISCUSSION THREAD

**Discussion by giscus** [SSWConsulting/SSW.Rules.Content#9691] [Rules Discussions]
# b455ea0e-0b1c-41e3-a97a-2e2315b20d08

Superannuation isn’t just savings, it’s a powerful investment. Maximize your future wealth with compound interest, strategic contributions, and smart management today!

https://www.ssw.com.au/rules/maximize-superannuation-benefits/

**Comment by BenDunkerleySSW** [comment]
Note: If you are on a VISA and you're thinking of leaving Australia, then there is a horrible 35% exit tax when you try to take your super out of Australia.  Otherwise, it stays in Australia until your retirement age.

**Comment by TobyChurches** [comment]
What a great rule!! I really like that when we put out this style of content!

Unfortunately super is not a area which I have previously placed much thought... But it is definitely on the list now

**Comment by jakebayliss** [comment]
More Super funds need to accept KiwiSaver transfers! -.-

**Comment by ricksu978** [comment]
It's a good idea to review your superannuation at least once a year, as regulations can change over time!

**Comment by awaltos** [comment]
Great vid Stephan 😄️ You are always able to deliver complex financial information in a clear and concise manner. This kind of thing usually bores me, but I'm always hooked with your content!

It's nice to get a reminder to look at Super as it's something that doesn't get enough thought. My priorities around Super are to get more of it so it is great to hear about these strategies. Thanks

**Comment by AntPolkanov** [comment]
Recently read an article [Are you being ripped off by super](https://www.morningstar.com.au/insights/retirement/257957/are-we-being-ripped-off-by-super) about super fees. I thought that 0.69% is a low interest, but after reading it I changed my mind.

**Comment by MarcusIrmscherSSW** [comment]
Great advice Stephan ⭐, young Dev's take notice.  
Harness that power of compounding 🪄

**Comment by sethdaily** [comment]
Super rule!

**Comment by isaaclombardssw** [comment]
Really interesting – helpful to give some of my attention to this kind of widely applicable life advice.

**Comment by stephanfakossw** [comment]
The priority is always fund performance. This includes reviewing your fund's past performance against a benchmark.

**Comment by steven0x51** [comment]
great finance 101 course in Super.

**Comment by sam-s-maher** [comment]
Appreciate the content and SSW's commitment to employees maximising their income, including other processes such as salary sacrificing for devices. Thanks Stephan!

**Comment by bahjatalaadelssw** [comment]
This is an important subject, I'm glad there are ethical super options available for investing.

**Comment by jeoffreyfischer** [comment]
> What are your priorities when it comes to choosing your Super?

I like it balanced, but am considering a more aggressive approach now

**Comment by uly1** [comment]
I've been paying extra into super for 10 years!!! I reocmmend everyone read Barefoot Investor :)

**Comment by Ben0189** [comment]
Thanks Stephan for sharing this information, I had never think of this before and will start looking into it 😊

**Comment by Lewkans** [comment]
Never focused on my super and just kept the same one that my first employer gave me, I should review it to see if there are better options

**Reply by Marxoz** [reply]
Extremely valuable information! Thank you Stephan!

**Comment by edkranz** [comment]
> 1. First think about something you learnt and what you think about the rules

The super comparison tool. It is a good rule, generally – people aren't aware enough about super.

    > 2. What are your priorities when it comes to choosing your Super?

📈📈📈📈📈

**Comment by tkapa** [comment]
I personally started becoming interested in my Super and Investments early. 

I think of myself as pretty good with money, so it's nice to hear that I was already doing a lot of this stuff

**Comment by wicksipedia** [comment]
In Australia, it’s awesome that this is enforced at a payroll level. Saving for the future can be tough, but this makes it a lot easier

**Comment by chrisschultzssw** [comment]
Great advice for Aussies starting out in their career (or anyone who hasn't thought about their super before).

**Comment by MattParkerDev** [comment]
Awesome practical advice! :)

**Comment by ncn-ssw** [comment]
Super is great if you cannot trust yourself with your own money.

Given that it's compulsory though, just have to prioritize low fees and decent long-term performance in your fund.

**Comment by Calinator444** [comment]
It's easy to put off worrying about your super because it'll only have implications pretty far down the line. It's important though.

**Comment by JackDevAU** [comment]
Will be coming back to this Rule!! Awesome stuff Stephan 💪 
Thanks for helping me make my retirement better ⭐

**Comment by SamBWagner** [comment]
**God tier video! ⚡️**

Fantastic Rule with an even better video! These are the style of Rules I want! #StephanForPresident

**Comment by PennyWalker** [comment]
Thanks for breaking down compounding interest in such an understandable way!

**Comment by amankumarrr** [comment]
Great video! It really made me reflect on my strategy and consider how to manage it more wisely for the future.

**Comment by ZenoWang1999** [comment]
Due to a lack of resources and energy to research individual stocks with potential, I generally opt to invest in sector-specific funds.

**Reply by tino-liu** [reply]
Be wise about your money

**Comment by babakamyljanovssw** [comment]
Informative rule as I know not much about Superannuation.

**Comment by jimmidier** [comment]
Great rule! It's surprised to see this topic being brought into CTF

**Comment by Marxoz** [comment]
Extremely valuable information! Thank you Stephan!

**Comment by Aibono1225** [comment]
Great rule and video!

**Comment by joshbermanssw** [comment]
Super is something I rarely considered or knew about... Thanks Stephan!

**Comment by suzanne-gibson** [comment]
As somebody who's just been onboarded this is a great time for me to evaluate my Super. Thanks for putting it top of mind.

**Comment by ozair-ashfaque** [comment]
Thanks for the rule 🙂

**Comment by drwharris** [comment]
Always pays to check if your super fund is performing. I didnt check for many years and and I reckon I lost thousands because they were underperforming quite badly.

**Comment by vladislav-kir** [comment]
Great video and the rule!

**Comment by bettybondoc** [comment]
Great rule! I had no idea you could salary sacrifice Super, let alone save money by doing it.

**Comment by danielmackay** [comment]
Great video and rule Stephan. Thanks for explaining everything so clearly. You make a very convincing argument to squirrel away a bit extra. :-)
